lib/mutant/ast/regexp/transformer.rb

```diff
@@ -12,7 +12,7 @@ class Transformer
         include AbstractType
 
         REGISTRY = Registry.new(
-          ->(type) { fail "No regexp transformer registered for: #{type}" }
+          ->(type) { } # fail "No regexp transformer registered for: #{type}" }
```

Comment:
actually my "failing test" [here](https://github.com/mbj/mutant/pull/1278/files#diff-16721c19d3917e959236dc238b8b4b202dee25963638b70deb7bb76070ac8b88R18-R21) probably won't fail with this disabled. i was trying to demonstrate that in another commit but I couldn't get CI to build separately. just try adding any not-explicitly-listed `\p{...}` group and it will fail mutant.